README.md

```
@@ -1,102 +1,179 @@
+# Unit tests
+
+## Necessary steps before testing
+
+First, do the steps from the [Install dependencies](#install-dependencies) section.
+
+The proceed with the following steps.
+
+### Requirements
+
+- [Requirements](#requirements)
+- CMLIB: https://github.com/cmakelib/cmakelib
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)